### ISSUE TITLE
Weapons with bonuses to destiny do not properly reflect that bonus in chat log

### ISSUE THREAD

**Issue by JediJer** [PlayersCommittee/gemp-swccg-public#573]
Noticed initially with Aurra Sing with Blaster Rifle

Example - 
target a Jedi character with defense value 6
draw 4 and 2 for weapon destinies
chat will show total weapon destiny = 6
defense value = 6
success

It should show total weapon destiny is 7

I have noticed this on a few other weapons that add to the total, most seem to be epps but there may be other weapon cards that are not properly reflecting the bonus to the draw, as well as modifiers from other game text sources (AOs, locations, characters, effects, etc.) It doesn't seem to be universal, as I have seen other weapons correctly include any appropriate bonuses when displaying the total in the chat.

While this isn't technically a bug (the end result is correctly played out), it can be confusing if watching the chat box

**Comment by EchoBaseTrooper** [comment]
Aurra's permanent weapon says "if total destiny + 1 > defense value. " The total weapon destiny is 6, but you compare "total destiny + 1" to the defense value. It can be confusing, but it is showing the correct value for total weapon destiny.

It might be a good idea to modify the text to make it more clear what is being compared, but weapons are the most complicated part of GEMP so a change to this is not very likely in the near future.

**Comment by BOTVHD** [comment]
I agree that an extra line, explaining that actual final calculation (for deciding if successful or not) would be a welcome feature in the log.  I guess this can be marked as an 'enhancement'?

This applies for other destiny draws too, I think.  For example, if I remember right, the log for playing He Hasn't Come Back Yet is not very clear about the final comparison (doesn't really show the effects of the +3 anywhere in the log to help the user understand why it was successful / unsuccessful).
